sites/all/themes/wille/assets/sass/component/_article_breol_news.scss

```diff
@@ -29,6 +29,9 @@
     margin-top: 20px;
     padding: 0 100px;
 
+    font-family: $font-regular;
```

Comment:
Won't this override the selected styles from the editor?

Reply:
Good point, but I think https://github.com/eReolen/base/blob/feature/ER-1122-lead-on-news/sites/all/themes/wille/assets/sass/base/_base.scss#L55-L57 saves the day.

Reply:
Alright then. Good work.